SignList.xml

```diff
@@ -0,0 +1,24 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <FirstParty Include="Xamarin.Android.*.dll" />
+    <FirstParty Include="Xamarin.AndroidX.*.dll" />
+    <FirstParty Include="Xamarin.Google.Android.Material.dll" />
+    <FirstParty Include="androidx-migrator.dll" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ThirdParty Include="ILRepack.dll" />
+    <ThirdParty Include="protobuf-net.dll" />
+    <ThirdParty Include="Newtonsoft.Json.dll" />
```

Comment:
`@(ThirdParty)` is for all the dlls that we included in the packages, but do not build here. This can be for random things from NuGet or pulled in from other places. These are just for UNSIGNED things.